packages/a/autoconf-archive/package.yml

```diff
@@ -1,9 +1,10 @@
 name       : autoconf-archive
-version    : 2019.01.06
-release    : 7
+version    : 2023.02.20
+release    : 8
 source     :
-    - https://ftp.gnu.org/gnu/autoconf-archive/autoconf-archive-2019.01.06.tar.xz : 17195c833098da79de5778ee90948f4c5d90ed1a0cf8391b4ab348e2ec511e3f
-license    : GPL-3.0-with-autoconf-exception
+    - http://ftp.wayne.edu/gnu/autoconf-archive/autoconf-archive-2023.02.20.tar.xz : 71d4048479ae28f1f5794619c3d72df9c01df49b1c628ef85fde37596dc31a33
+homepage   : https://www.gnu.org/software/autoconf-archive/
+license    : GPL-3.0-or-later WITH Autoconf-exception-3.0
```

Comment:
According to [SPDX](https://spdx.org/licenses/GPL-3.0-with-autoconf-exception.html) this license identifier is correct

Reply:
Yes, but deprecated isn't it?. 

> DEPRECATED: Use license expression including main license, "WITH" operator, and identifier: Autoconf-exception-3.0

The new one should be the correct one. I am happy to change it back.

Reply:
No, it isn't: It should be `GPL-3.0-with-autoconf-exception`